converter/convert_test.go

```diff
@@ -33,59 +33,68 @@ func TestConvertFromFixedUpBundleToOCI(t *testing.T) {
 	assert.DeepEqual(t, expected, actual)
 
 	// Nil maintainers does not add annotation
-	src = test.MakeTestBundle()
+	src = tests.MakeTestBundle()
 	src.Maintainers = nil
 	actual, err = ConvertBundleToOCIIndex(src, named, bundleConfigDescriptor)
 	assert.NilError(t, err)
 	_, hasMaintainers := actual.Annotations[ocischemav1.AnnotationAuthors]
 	assert.Assert(t, !hasMaintainers)
 
 	// Nil keywords does not add annotation
-	src = test.MakeTestBundle()
+	src = tests.MakeTestBundle()
 	src.Keywords = nil
 	actual, err = ConvertBundleToOCIIndex(src, named, bundleConfigDescriptor)
 	assert.NilError(t, err)
 	_, hasKeywords := actual.Annotations[CNABKeywordsAnnotation]
 	assert.Assert(t, !hasKeywords)
 
 	// Multiple invocation images is not supported
-	src = test.MakeTestBundle()
+	src = tests.MakeTestBundle()
 	src.InvocationImages = append(src.InvocationImages, src.InvocationImages[0])
 	_, err = ConvertBundleToOCIIndex(src, named, bundleConfigDescriptor)
 	assert.ErrorContains(t, err, "only one invocation image supported")
 
 	// Invalid media type
-	src = test.MakeTestBundle()
+	src = tests.MakeTestBundle()
 	src.InvocationImages[0].MediaType = "some-invalid-mediatype"
 	_, err = ConvertBundleToOCIIndex(src, named, bundleConfigDescriptor)
-	assert.ErrorContains(t, err, "unsupported media type \"some-invalid-mediatype\" for image \"my.registry/namespace/my-app@sha256:d59a1aa7866258751a261bae525a1842c7ff0662d4f34a355d5f36826abc0341\"")
+	assert.ErrorContains(t, err, "unsupported media type "+
```

Comment:
Maybe use backslash notation instead? I know it won't work with 80 columns but it will remove the `\"`

Reply:
Done